src/reference/teststreams/index.pug

```diff
@@ -0,0 +1,58 @@
+extends ../reference_layout.pug
+
+block title
+    | Test Streams | Mixer Developers
+
+block navbar
+    .navbar-wrapper
+        .container
+            nav.navbar.navbar-inverse: +navbar()
+            h1 Test Streams
+block menu
+    .hidden-print.rest-sidebar(role='complementary')
+        nav
+            ul.nav.nav-stacked
+                li: a(href='#introduction') Introduction
+                li: a(href='#applying') Applying for Test Stream Access
+                li: a(href='#enabling') Enabling Test Stream Mode
+                li: a(href='#controlling') Controlling Access
+                li: a(href='#best-practices') Best Practices
+block reference
+
+    h2#introduction Introduction
+
+    p Test streams allow you to broadcast your content in private. This means if you are working on an unannounced game or just want to try out some new streaming settings then you can use test streams to broadcast without anyone being able to see your content or broadcast.
+
+    h2#applying Applying for Test Stream Access
+    p Test Streams are is currently released to a limited audience. You can apply to use test streams with the following steps:
+    ol
```

Comment:
Can you throw in a link to https://mixer.com/lab/teststreams for the lazy folks?